openquake/hazardlib/gsim/mgmpe/nrcan15_site_term.py

```diff
@@ -56,6 +56,10 @@ def __init__(self, gmpe_name):
             tmps = '{:s} does not use vs30 nor a defined reference velocity'
             msg = tmps.format(str(self.gmpe))
             raise AttributeError(msg)
+        if not (hasattr(self.gmpe, 'REQUIRES_SITES_PARAMETERS')):
+            self.REQUIRES_SITES_PARAMETERS = set(('vs30'))
```

Comment:
`(hasattr(self.gmpe, 'REQUIRES_SITES_PARAMETERS'))`

you do not need external parenthesis here.

Reply:
`set(('vs30'))` is wrong, you want  `set(('vs30',))` or  `set(['vs30'])` or `{'vs30'}`.